docs/ko/translation-guide.md

```diff
@@ -267,32 +267,32 @@ A value of 0.01 was used for the value to ramp down to in the last function rath
 | 용어 | 번역 | 참고 링크 |
 | --- | --- | --- |
 | Accessibility concerns | 접근성 고려사항 | |
-| Advantages | 장점 | [링크](https://developer.mozilla.org/en-US/docs/Games/Introduction_to_HTML5_Game_Development) |
-| Aliasing | 별칭 | [링크](https://github.com/mdn/translated-content/pull/1779/files) |
-| Brief history | 간략한 역사 | [링크](https://developer.mozilla.org/ko/docs/Glossary/HTML#%EA%B0%84%EB%9E%B5%ED%95%9C_%EC%97%AD%EC%82%AC) |
-| Browser compatibility | 브라우저 호환성 | [링크](https://github.com/mdn/translated-content/pull/1779/files) |
-| Browser support | 브라우저 지원 | [링크](http://localhost:5042/en-us/docs/Web/API/Web_Audio_API/Using_IIR_filters#browser_support) |
+| Advantages | 장점 | <https://developer.mozilla.org/en-US/docs/Games/Introduction_to_HTML5_Game_Development> |
+| Aliasing | 별칭 | <https://github.com/mdn/translated-content/pull/1779/files> |
+| Brief history | 간략한 역사 | <https://developer.mozilla.org/ko/docs/Glossary/HTML#%EA%B0%84%EB%9E%B5%ED%95%9C_%EC%97%AD%EC%82%AC> |
+| Browser compatibility | 브라우저 호환성 | <https://github.com/mdn/translated-content/pull/1779/files> |
+| Browser support | 브라우저 지원 | <http://localhost:5042/en-us/docs/Web/API/Web_Audio_API/Using_IIR_filters#browser_support> |
```

Comment:
`localhost` link here

Reply:
I'm not sure if it's intentional or not, I didn't change those links, just changed them to bare versions

Reply:
localhost is a mistake. It is correct that they are replaced with actual commercially distributed links.

Reply:
```suggestion
| Browser support | 브라우저 지원 | <https://developer.mozilla.org/en-us/docs/Web/API/Web_Audio_API/Using_IIR_filters#browser_support> |
```